src/templates/candidate.module.scss

```diff
@@ -23,7 +23,7 @@
 
 .aboutLinks {
   display: flex;
-  flex-wrap: wrap;
+  flex-direction: column;
```

Comment:
I think the links are supposed to display in a row on larger screens, and then I set flex-wrap to deal with smaller screen sizes. I could do the same with a media query, but I thought this looked slightly better for the in-between sizes (instead of awkwardly wrapping the text)